src/core/scene/a-scene.js

```diff
@@ -72,13 +80,12 @@ var AScene = module.exports = registerElement('a-scene', {
 
     attachedCallback: {
       value: function () {
-        initFullscreen(this);
         initMetaTags(this);
         initWakelock(this);
 
         window.addEventListener('load', this.resize.bind(this));
```

Comment:
why don't we set an intermediate variable (e.g., `var resize = this.resize.bind(this);`)?